src/Voron/Constants.cs

```diff
@@ -91,7 +91,9 @@ static Tree()
         public static readonly Slice DatabaseFilenameSlice;
         public const string DatabaseRecordFileNameInBackup = "___database_record___";
 
-        public const int DefaultMaxLogLengthBeforeCompaction = 64; //how much entries in log to keep before compacting it into snapshot       
+        public const int DefaultMaxLogLengthBeforeCompaction = 64; //how much entries in log to keep before compacting it into snapshot
+
+        public const string SkippedStreamsDetailsName = "Stream details summary info";
```

Comment:
In this file we have constants that have fundamental / global meaning for the Voron itself. The one that you added is very specific to the storage report generation - which isn't a fundamental functionality.

I think a way better candidate would be to put it in `StorageReportGenerator`. Especially that we use it only there

Reply:
done